tower/src/buffer/service.rs

```diff
@@ -87,12 +107,18 @@ where
     type Future = ResponseFuture<T::Future>;
 
     fn poll_ready(&mut self, cx: &mut Context<'_>) -> Poll<Result<(), Self::Error>> {
-        // If the inner service has errored, then we error here.
-        if let Err(_) = ready!(self.tx.poll_ready(cx)) {
-            Poll::Ready(Err(self.get_worker_error()))
-        } else {
-            Poll::Ready(Ok(()))
+        tracing::info!("poll");
```

Comment:
I had issues with this test previously, but also by the looks of these tracing statements you did too? Probably should remove these :)

Reply:
oh whoops, didn't mean to leave those in!